CONTRIBUTING.md

```diff
@@ -106,11 +106,11 @@ I personally do not like something that requires so many configurations before y
 
 ## Tools
 
-- Node.js >= 14
-- NPM >= 8.5
-- Git
-- IDE that supports ESLint and EditorConfig (I am using IntelliJ IDEA)
-- A SQLite GUI tool (SQLite Expert Personal is suggested)
+- [`Node.js`](https://nodejs.org/de) >= 14
+- [`npm`](https://www.npmjs.com/) >= 8.5
+- [`git`](https://git-scm.com/)
+- IDE that supports [`ESLint`](https://eslint.org/) and EditorConfig (I am using [`IntelliJ IDEA`](https://www.jetbrains.com/de-de/idea/))
+- A SQLite GUI tool (f.ex. [`SQLite Expert Personal`](https://www.sqliteexpert.com/download.html))
```

Comment:
Can I suggest to promote an open source tool instead? DBeaver?

Reply:
Why? What makes it better?

Reply:
Because:
- a new comer will probably prefer a free tool (community edition of DBeaver)
- UK is an open source tool too, my personal feeling is that the open source community should prefer self-promote user source tools 🤷🏼‍♂️

Reply:
- But Sqlite Expert also as a free tier, just as [DBeaver](https://dbeaver.com/edition/) has a paid tier.
  Given that the free tier of DBeaver is quite difficult to find, I would not consider this to be a significant factor.
- As I have not used either tool, I cannot make a judgement about the merits of both tools.
  ⇒ Louis can decide whether he wants to add this suggestion or not

```suggestion
- A SQLite GUI tool (f.ex. [`SQLite Expert Personal`](https://www.sqliteexpert.com/download.html) or [`DBeaver Communtiy`](https://dbeaver.io/download/))
```

Reply:
It's OK.